### PR TITLE
IZPACK-1731 & IZPACK-1734

### PR DESCRIPTION
**Pull request by hiteshbosamiya**
[IZPACK-1731] UserPathPanel.intro message string does not work for UserPathPanel, no need for extendedIntro key
[IZPACK-1734] Missing properties implemenatation for UserPathPanel

[IZPACK-1731]: https://izpack.atlassian.net/browse/IZPACK-1731?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IZPACK-1734]: https://izpack.atlassian.net/browse/IZPACK-1734?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathConsolePanel.java [hunk 1]
```diff
@@ -43,29 +46,16 @@
  */
 public class UserPathConsolePanel extends AbstractConsolePanel
 {
-    public static final String PATH_VARIABLE;
-    public static final String PATH_PACK_DEPENDS;
-    public static final String PATH_ELEMENT;
-    public static final String USER_PATH_INFO;
-    public static final String USER_PATH_NODIR;
-    public static final String USER_PATH_EXISTS;
+    public static final String PATH_VARIABLE = UserPathPanel.PATH_VARIABLE_NAME;
```
Comment (reinhapa) [inline izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathConsolePanel.java:49]: Why not using a static import for this constant?

File: izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanelAutomationHelper.java [hunk 2]
```diff
@@ -44,13 +44,13 @@ public UserPathPanelAutomationHelper()
     public void createInstallationRecord(InstallData idata, IXMLElement panelRoot)
     {
         // Installation path markup
-        IXMLElement ipath = new XMLElementImpl(UserPathPanel.pathElementName, panelRoot);
+        IXMLElement ipath = new XMLElementImpl(UserPathPanel.PATH_ELEMENT_NAME, panelRoot);
```
Comment (reinhapa) [inline izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanelAutomationHelper.java:47]: Why not using a static import for this constant here and for the following usages?

File: izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanel.java [hunk 3]
```diff
@@ -98,16 +98,16 @@ public void panelActivate()
         }
         if (skip)
         {
-            logger.fine(UserPathPanel.class.getSimpleName() + " will not be shown");
+            logger.fine(UserPathPanel.NAME + " will not be shown");
```
Comment (reinhapa) [inline izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanel.java:101]: Static import for this constant?

**Review by reinhapa — CHANGES_REQUESTED** [review]

File: izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathConsolePanel.java (hunk 1 above)
Comment (hiteshbosamiya) [reply izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathConsolePanel.java:49]: Done, using static import now.

File: izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanelAutomationHelper.java (hunk 2 above)
Comment (hiteshbosamiya) [reply izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanelAutomationHelper.java:47]: Done, using static import now.

File: izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanel.java (hunk 3 above)
Comment (hiteshbosamiya) [reply izpack-panel/src/main/java/com/izforge/izpack/panels/userpath/UserPathPanel.java:101]: This is within the same class, renamed the variable as PANEL_NAME and using only the variable name.

**Review by akuhtz — APPROVED** [review]

**Pull request merged**